demisto_sdk/commands/common/tools.py

```diff
@@ -3665,3 +3665,42 @@ def parse_marketplace_kwargs(kwargs: Dict[str, Any]) -> MarketplaceVersions:
         "neither marketplace nor is_xsiam provided, using default marketplace=XSOAR"
     )
     return MarketplaceVersions.XSOAR  # default
+
+
+def get_api_module_from_graph(changed_api_modules: set, graph=None) -> List:
+    if changed_api_modules:
+        dependent_items = []
+        if not graph:
+            from demisto_sdk.commands.content_graph.interface.neo4j.neo4j_graph import (
+                Neo4jContentGraphInterface,
+            )
+
+            graph = Neo4jContentGraphInterface(should_update=True)
```

Comment:
Notice that the `graph` driver should be closed, so I don't think it's a good idea to create it here (without closing it).
Let's add the graph to the `UpdateRNManager` object, and will run `graph.close()` when it finishes.